titus-common-ext/kube/src/main/java/com/netflix/titus/ext/kube/clustermembership/connector/KubeClusterMembershipConnectorComponent.java

```diff
@@ -70,7 +69,7 @@ public ApiClient getApiClient(KubeConnectorConfiguration configuration) {
             client = Config.fromUrl(kubeApiServerUri);
         }
 
-        client.getHttpClient().setReadTimeout(0, TimeUnit.SECONDS); // infinite timeout
+        client.setReadTimeout(0); // infinite timeout
```

Comment:
can you see if this is still needed?

Reply:
I do not know why it was set like that. I am just keeping the original behavior.
Even if not needed, I do not want to remove it now.